jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -914,8 +935,8 @@ Object runNativeMethod(Addressable downcallAddr, SegmentAllocator segmtAllocator
 					returnStateMemBase,
 					/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
 					/*[IF JAVA_SPEC_VERSION >= 22]*/
-					(info != null) ? info.bases : null,
-					(info != null) ? info.offsets : null,
+					(info != null) ? info.getBases() : null,
```

Comment:
Is. it possible that `info == null` here? if not you can just pass `bases` and `offsets` without the null check